src/crafting.cpp

```diff
@@ -404,8 +404,10 @@ bool player::check_eligible_containers_for_crafting( const recipe &rec, int batc
         }
 
         if( charges_to_store > 0 ) {
-            popup( _( "You don't have anything to store %s in!" ), prod.tname() );
-            return false;
+            if( !query_yn( _( ( "You don't have anything in which to store %s and may have to pour it out or consume it as soon as it is prepared! Proceed?" ),
```

Comment:
These extra parenthesis don't serve a purpose.
```suggestion
            if( !query_yn( _(  "You don't have anything in which to store %s and may have to pour it out or consume it as soon as it is prepared! Proceed?",
```

Reply:
This change is making all of the tests fail I think.

Reply:
No, it fails to compile anyways.
You need to string format this, and those parenthesis are still pointless.

Reply:
Trying again...

Reply:
You need to call `string_format` on it, so that you can have the name of the product replace `%s`. query_yn automatically does this, but  `_()` does not.

Reply:
Like this?

Reply:
query_yn already does string formatting.
https://github.com/CleverRaven/Cataclysm-DDA/blob/cf4ce7d691d9acb58cfb7b667e2abeb0f60fe595/src/output.h#L372-L377
And `_()` takes one argument.
So it should be:
```cpp
query_yn( _( "You don't have anything in which to store %s and may have to pour it out or consume it as soon as it is prepared! Proceed?" ), prod.tname() )
```